.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,20 @@
+name: Checkstyle
+
+on: pull_request
+
+jobs:
+  checkstyle_job:
+    runs-on: ubuntu-latest
+    name: Checkstyle
+    steps:
+      - name: Checkout Branch
+        uses: actions/checkout@v2
+      - name: Run Checkstyle
+        uses: nikitasavinov/checkstyle-action@master
+        with:
+          level: error
+          reporter: 'github-pr-check'
+          checkstyle_config: "/home/runner/work/chasm/chasm/config/checkstyle/checkstyle.xml"
```

Comment:
Can this be a relative path?
Not a big deal if it can't, but would be cleaner.

Reply:
No, it has to be specified from the root folder.

Reply:
You can use the environment variable `GITHUB_WORKSPACE`. See https://docs.github.com/en/actions/learn-github-actions/environment-variables.

Reply:
Didn't think about this, thanks!